src/utils/errorHandler.js

```diff
@@ -0,0 +1,66 @@
+import saveError from "./ErrorHistory";
```

Comment:
Se o problema não for apenas no meu computador, provavelmente tem algo a ver com o `require` aqui